code/modules/shuttle/custom_shuttle/shuttle_creator_eye.dm

```diff
@@ -0,0 +1,52 @@
+//===============Camera Eye================
+/mob/camera/ai_eye/remote/shuttle_creation
+	name = "shuttle holo-drone"
+	/*icon = 'icons/obj/mining.dmi'*/
+	icon_state = "construction_drone"
+	visible_icon =  FALSE
+	acceleration = 0
+	var/turf/source_turf
+	var/max_range = 12
+
+/mob/camera/ai_eye/remote/shuttle_creation/Initialize(mapload)
+	. = ..()
+	setLoc(get_turf(source_turf))
```

Comment:
FYI, this is one of those situations where you don't have to change the casing of the var name, because it was already here and it isn't your doing.